src/media/js/error_messages.js

```diff
@@ -0,0 +1,26 @@
+/*
+    Error codes and their associated messages.
+*/
+define('error_messages', ['core/l10n'], function(l10n) {
+    'use strict';
+    var gettext = l10n.gettext;
+
+    return {
```

Comment:
might be better for this to take generic status codes so we don't have to write a message for each error